packages/ai-jsx/src/lib/openai.tsx

```diff
@@ -2,8 +2,6 @@
  * This module provides interfaces to OpenAI's various models.
  * @packageDocumentation
  */
-
-import GPT3Tokenizer from 'gpt3-tokenizer';
```

Comment:
I'm totally fine with this, but can you share the links you found as to why this switch was necessary? Just to satisfy my curiosity & no need to spend more than 20 seconds looking.

Reply:
From https://github.com/openai/openai-cookbook/blob/main/examples/How_to_count_tokens_with_tiktoken.ipynb the implication is that gpt-3.5-turbo/gpt-4 use a different tokenizer than gpt-3.

In practice how much does it matter for simple estimations? 🤷 But since JS ports of tiktoken exist (nominally the source of truth) exist I opted to switch.

Reply:
Cool. Looks like Anthropic has their own thing fwiw https://www.npmjs.com/package/@anthropic-ai/tokenizer